src/App.jsx

```diff
@@ -1,104 +1,30 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
+
+function getUserById(userId) {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
+
+function getCommentsByPostId(postId) {
+  return (
+    commentsFromServer.filter(comment => comment.postId === postId) || null
+  );
+}
+
+export const posts = postsFromServer.map(post => ({
+  ...post,
+  user: getUserById(post.userId),
+  comments: getCommentsByPostId(post.Id),
```

Comment:
Maybe you should channge post.Id to post.id here

Reply:
I've been looking for this error for a long time